src/components/flat-table/flat-table.component.js

```diff
@@ -102,6 +101,8 @@ FlatTable.propTypes = {
   hasStickyFooter: PropTypes.bool,
   /** Set the height of the table */
   height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  /** Set the min-height of the table */
```

Comment:
Worth noting in the comment that the string can be any valid CSS string, and a number will be converted to a pixel value - it'll be less confusing for users in the prop table

Reply:
Yeah good point will add that in thanks